packages/react-components/src/components/styles/Captions.style.ts

```diff
@@ -35,7 +35,7 @@ export const captionClassName = mergeStyles({
 export const captionContainerClassName = mergeStyles({
   marginTop: _pxToRem(6),
   marginBottom: _pxToRem(6),
-  overflowAnchor: 'none'
+  overflowAnchor: 'auto'
```

Comment:
Does this then track the message by the key and stops the overflow from scrolling when there is a new caption?

Reply:
Right! It tracks messages by the dom element itself, and the key is to ensure react keeps all dom elements to be same instances, instead of re-creating them again and again, so that browser can track them well